task/rhacs-deployment-check/3.71/README.md

```diff
@@ -53,20 +106,29 @@ kubectl create secret generic rox-central-endpoint \
     workspaces:
     - name: source
       workspace: shared-workspace
+    - name: rox-api-token-auth
+      secret:
+        secretName: rox-api-token
     params:
     - name: deployment
       value: $(params.deployment)
+    - name: rox_central_endpoint
+      value: central.stackrox.svc:443
+    - name: rox_token_file
+      value: /rox-api-token-auth/rox_api_token
     runAfter:
     - fetch-repository
 ```
 
 **Samples:**
 
-* [secrets.yaml](samples/secrets.yaml) example secret
-* [pipeline.yaml](samples/pipeline.yaml) demonstrates use in a pipeline.
-* [pipelinerun.yaml](samples/pipelinerun.yaml) demonstrates use in a pipelinerun.
+* [secrets.yaml](samples/with-api-token/secrets.yaml) example secret
+* [pipeline.yaml](samples/with-api-token/pipeline.yaml) demonstrates use in a pipeline.
+* [pipelinerun.yaml](samples/with-api-token/pipelinerun.yaml) demonstrates use in a pipelinerun.
 
 # Known Issues
 
 * Skipping TLS Verify is currently required. TLS trust bundle not working for quay.io etc.
-* If the namespace value is not found in the deployment manifest any RHACS policies which are scoped to specific namespaces will not be matched.
\ No newline at end of file
+* If the namespace value is not found in the deployment manifest any RHACS policies which are scoped to specific namespaces will not be matched.
+
+* Version of roxctl should maintain compatibility with Central API. Maximum allowable version drift is unknown.
```

Comment:
Whats the point of this here? This doesn't really bring much besides confusion IMO and its more of a best practice really.